keps/sig-architecture/4330-compatibility-versions/README.md

```diff
@@ -610,21 +623,25 @@ The storage version of each group-resource is the newest
 
 ### API availability
 
-Similar to feature flags, all APIs group-versions declarations will be modified
-to track which Kubernetes version the API group-versions are introduced (or
-removed) at.
+Ideally, similar to feature flags, all APIs group-version-resource declarations should be modified
+to track which Kubernetes version the GVRs are introduced (or
+removed) at: if an API is introduced after (or removed before) the emulation version, it should not be available at the emulation version. But in practice, that would make code changes of API graduation that wants to use newer APIs intractable. 
+For example, if a controller is switching from Beta to GA API at the binary version, a full emulation would mean the whole controller code needs to be
```

Comment:
My understanding is that we're doing this specifically for controllers.  Are there other use cases or should we simplify the wording here and just state up front that we're doing this to support controllers specifically?